src/content/docs/ai-monitoring/intro-to-ai-monitoring.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Introduction to AI monitoring'
+metaDescription: 'AI monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import aiTraceViewIntroPage from 'images/ai_screenshot-full_trace-view-intro-page.webp'
+
+import aiAIResponsesOverview from 'images/ai_screenshot-full_AI-responses-overview.webp'
+
+When New Relic talks about AI, we're referring to the layer of your environment that uses a large language model (LLM) to generate a response when it receives an end user prompt. Your app contains an AI assistant, but behind the AI are libraries, cost implications, vector databases, and new processes that can be monitored.   
+
+<img
+    title="Trace waterfall for AI monitoring"
+    alt="A screenshot that shows the trace waterfall page for an individual AI response"
+    src={aiTraceViewIntroPage}
+/>
+
+<figcaption>
+    Go to **[one.newrelic.com](https://one.newrelic.com) > AI monitoring > AI responses**, then select a response row. 
+</figcaption>
+
+With AI monitoring, you can evaluate the performance of your AI by instrumenting your app with one of our APM agents. AI monitoring collects trace and span data about how your model generated a response, tracks performance metrics like latency and error rate, and gives you more oversight into your model's token usage. 
+
+## What is AI Monitoring? [#what-is-ai-monitoring]
+
+When your AI receives a prompt, an agent traces how the response generated through external LLMs, vector stores, and event data. From the UI, you can monitor performance trends through app metrics like uptime, latency, and error rate. With AI monitoring, you can:
+
+* **Evaluate errors and bugs in your code.** Use the responses table to identify AI responses that contain errors. Dig into the methods and calls your AI-powered app makes when generating a response. 
+* **Track token usage.** Did your prompt engineers make a recent change that affected cost? Use the AI response page to track tokens and keep costs down.
+* **Compare models.** If you're using different models in different environments, compare their performance against how much they cost.
```

Comment:
dollars or computational costs?

Reply:
I'd like these three bullets to feel more equal/equivalent. For example, the question in bullet two feels kind of interesting and engaging, but it also feels odd as a one off. 

If I were someone deep into AI, I might know things like: 

Evaluate errors: What kind of errors? Do I have granular control over what errors are important to me and what aren't? Does this include warnings as well?
Track token usage: At what level are tokens tracked? How much detail am I going to get?
Compare models: What kind of comparisons are used? Is just how much computational resources they use or is it other things too, like efficiency or response speed?